From 6e457e14dcf0960926109c2f89766e893f79b5da Mon Sep 17 00:00:00 2001
From: caihaomin <caihaomin@huawei.com>
Date: Wed, 24 Oct 2018 14:56:01 +0800
Subject: [PATCH 61/94] runc-17: change golang build version to make
 obs happy

reason:change golang build version to make obs happy

Change-Id: Ie4b210056b34611bfb37aa6ea8510540f2c79305
Signed-off-by: caihaomin <caihaomin@huawei.com>
---
 script/runc-euleros.spec | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/script/runc-euleros.spec b/script/runc-euleros.spec
index 5318ec2..048ea86 100644
--- a/script/runc-euleros.spec
+++ b/script/runc-euleros.spec
@@ -2,7 +2,7 @@
 
 Name: docker-runc
 Version: 1.0.0.rc3
-Release: 3%{?dist}
+Release: 4%{?dist}
 Summary: runc is a CLI tool for spawning and running containers according to the OCF specification
 
 License: ASL 2.0
@@ -12,7 +12,7 @@ URL: https://www.opencontainers.org/
 Vendor: OCI
 Packager: OCI 
 
-BuildRequires: golang == 1.8.3
+BuildRequires: golang >= 1.8.3
 BuildRequires: glibc-static
 BuildRequires: make
 BuildRequires: libseccomp-devel 
-- 
2.7.4.3

